### PR TITLE
Translate team list

### PR DESCRIPTION
**Pull request by DEVTomatoCake**
## Type of PR

- [x] Translation update / creation
- [ ] Original file creation
- [ ] Original file content (not typo) update
- [x] Original file typo update

## Checks

- [x] Yes, I have copied the updated section to all translations of this file manually.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+* Füge Beschreibungen zu Rollen hinzu und überschreibe Rollennamen.
+
+Hier ist ein Beispiel (du kannst das Aussehen anpassen):
+
+![](@site/docs/assets/custom-bot/modules/team-list/example.png)
+
+## Einrichtung {#setup}
+
+1. Erstelle einen Kanal, in welchem der Bot die Teamliste senden soll. Dieser Kanal muss leer sein, und der Bot muss die Berechtigung "Kanal ansehen", "Nachrichtenverlauf anzeigen" und "Nachricht senden" haben.
+2. Öffne die [Modulkonfiguration](https://scnx.app/glink?page=bot/configuration?file=team-list%7Cconfig) und [konfiguriere das Modul](#configuration).
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md:20]:
```suggestion
2. Öffne die [Modulkonfiguration](https://scnx.app/de/glink?page=bot/configuration?file=team-list%7Cconfig) und [konfiguriere das Modul](#configuration).
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+|----------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Kanal                          | Dies ist der Kanal, in den die Teamliste gesendet wird. Wenn der Bot bereits eine Nachricht davor gesendet hat, wird die vorhandene Nachricht zur Teamliste bearbeitet. Wir empfehlen einen leeren Kanal.                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md:36]:
```suggestion
| Kanal                          | Dies ist der Kanal, in den die Teamliste gesendet wird. Wenn der Bot zuvor bereits eine Nachricht gesendet hat, wird die vorhandene Nachricht zur Teamliste bearbeitet. Wir empfehlen einen leeren Kanal.                                                                                                                                                                                                                                                                                                                                                                           |
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		<li>Versuche, die betroffene Rolle in kleinere Teile aufzuteilen.</li>
+	</ul>
+</details>
+
+<details>
+	<summary>Ein Mitglied fehlt in der Teamliste / Die Teamliste ist nicht aktuell</summary>
+	<ul>
+		<li>Stelle sicher, dass der ausgewählte Kanal leer ist.</li>
+		<li>Stelle sicher, dass der Bot die Berechtigungen "Kanal ansehen", "Nachrichtenverlauf anzeigen" und "Nachrichten senden" in dem Kanal hat.</li>
+		<li>Stelle sicher, dass du mindestens 15 Minuten lang gewartet hast, da das das Intervall ist, in dem sich das Embed aktualisiert.</li>
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md:71]:
```suggestion
		<li>Stelle sicher, dass du mindestens 15 Minuten lang gewartet hast, da dies das Intervall ist, in dem sich das Embed aktualisiert.</li>
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+## Verwendung {#usage}
+
+Mitglieder werden anhand deiner [konfigurierten Rollen](#configuration) angezeigt. Du kannst Mitglieder nicht manuell hinzufügen oder entfernen.
+
+Das Teamlisten-Embed wird automatisch alle 15 Minuten aktualisiert - von deiner Seite aus ist keine Aktion notwendig. Das bedeutet auch, dass Änderungen (wie das Entfernen einer Team-Rolle von einem Mitglied) bis zu 15 Minuten dauern können, um angezeigt zu werden.
+
+## Konfiguration {#configuration}
+
+Diese Konfigurationdatei erlaubt dir, einzustellen, welche Rollen angezeigt werden und wie das Embed aussehen soll.
+Öffne sie in deinem [Dashboard](https://scnx.app/glink?page=bot/configuration?file=team-list%7Cconfig).
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md:32]:
```suggestion
Öffne sie in deinem [Dashboard](https://scnx.app/de/glink?page=bot/configuration?file=team-list%7Cconfig).
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+| Embed                            | Dies sind Einstellungen, die das Aussehen deines Embeds auf Discord beeinflussen.<br/>`title`: Dies ist der Titel des Embeds, welcher als erstes im Embed angezeigt wird.<br/>`description` (optional): Dies ist der Text, der unter dem Titel im Embed angezeigt wird.<br/>`color`: Dies ist die Farbe des Embeds (verwende eine [HEX-Farbe oder einen unterstützten Farbwert](./../../additional-features#embed-colors))<br/>`thumbnail-url` / `img-url` (optional): URLs zum Anzeigen von Bildern im Embed. Du kannst den [Dateien-Speicher](https://scnx.app/user/files) zum Hochladen von einem Bild verwenden und die Bild-URL kopieren.|
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-list.md:39]:
```suggestion
| Embed                            | Dies sind Einstellungen, die das Aussehen deines Embeds auf Discord beeinflussen.<br/>`title`: Dies ist der Titel des Embeds, welcher als erstes im Embed angezeigt wird.<br/>`description` (optional): Dies ist der Text, der unter dem Titel im Embed angezeigt wird.<br/>`color`: Dies ist die Farbe des Embeds (verwende eine [HEX-Farbe oder einen unterstützten Farbwert](./../../additional-features#embed-colors))<br/>`thumbnail-url` / `img-url` (optional): URLs zum Anzeigen von Bildern im Embed. Du kannst den [Dateien-Speicher](https://scnx.app/de/user/files) zum Hochladen von einem Bild verwenden und die Bild-URL kopieren.|
```

**Review by SCDerox — CHANGES_REQUESTED** [review]

**Review by Fridolin012 — CHANGES_REQUESTED** [review]

**Review by Fridolin012 — APPROVED** [review]

**Comment by DEVTomatoCake** [comment]
Kommentar 2

**Comment by SCDerox** [comment]
Thanks for getting back to us and sorry for the delay!

Thanks for your contribution - you have received a payout as part of our [Open Source Developer Program](https://faq.scnx.app/open-source-developer-pool/). You can find details in your SCNX organisation payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on [scnx.app/help](https://scnx.app/help).
Cheers ✌️

**Pull request merged**